### PR TITLE
Options

### LINKED ISSUE #6: Support libraw options

### PR DESCRIPTION
**Pull request by SamWhited**
Adds high level support for basic image options.

As we add more, this will give us a way to normalize options (eg. all percentages could be ints from 0–100 or floats from 0–1) that may be inconsistent in libraw. This also gives us a way to document options better than the libraw or dcraw documentation, and gives us a single interface for setting all options; the dcraw emulation options, and those that libraw defines itself.

Fixes #6

Tests forthcoming.

### CODE REVIEW

File: rawkit/options.py [hunk 1]
```diff
@@ -0,0 +1,405 @@
...
+"""Adobe RGB (1998) D65"""
+wide_gammut_rgb = 3
+"""Wide Gamut RGB D65"""
+kodak_prophoto_rgb = 4
+"""Kodak ProPhoto RGB D65"""
+xyz = 5
+"""XYZ colorspace"""
+
+
+class WhiteBalance(object):
```
Comment (campaul) [inline rawkit/options.py]: This should probably be a NamedTuple.

File: rawkit/options.py [hunk 2]
```diff
@@ -0,0 +1,405 @@
+""":mod:`rawkit.options` --- High level options for processing raw files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+"""
+
+import ctypes
+
+CLIP = 0
```
Comment (campaul) [inline rawkit/options.py]: There should be some encapsulation of constants like this so we don't completely pollute the `rawkit.options` namespace.

File: rawkit/options.py (hunk 1 above)
Comment (SamWhited) [inline rawkit/options.py]: Done.

**Comment by SamWhited** [comment]
Squashed and merged.

**Pull request closed without merging**